### DISCUSSION TITLE
security streaming key

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3865] [Q&A] [answered]
Dear Owncast community, I have a question for you.
I have OC installed on a VPS as it comes by default without any extra configuration.
A few days ago, reviewing the panel, I found messages in the logs (invalid streaming key; rejecting incoming stream from....) (see image)
I think this is an attempt by bad people to try to access our broadcast.
My question is how can I reinforce security in these cases? Are these attempts normal? I would like you to support me with some advice

Greetings

![Owncast-Admin](https://github.com/user-attachments/assets/43a9ea96-c873-4791-b807-3859a8fba98b)

**Comment by gabek** [comment, marked as answer]
Hi there,

Just like any service on the public internet, you're likely to get probed by mass scans. They might not even realize it's an RTMP service they're touching.

If you'd like to secure this further, you'd treat it just like any other service. You could limit inbound connections on your firewall to just your IP address, or block all incoming RTMP traffic and setup a VPN, etc.

But there's nothing Owncast specific in this case, this is a side effect of being on the internet.

**Reply by fsan2val** [reply]
Perfect, thank you very much as always for clarifying my concerns.
Greetings and thanks again
